onnxscript/_internal/param_manipulation.py

```diff
@@ -70,6 +70,9 @@ def separate_input_attributes_from_arguments(
             else:
                 continue
         elif param.required:
-            raise TypeError(f"Required input '{param}' was not provided")
+            # comment this line becuase the Optional input will get error here
+            # raise TypeError(f"Required input '{param}' was not provided")
+            print(f"Required input '{param}' was not provided")
+
```

Comment:
```suggestion
            raise TypeError(f"Required input '{param}' was not provided")
```

let me try what happens here